flink-ml-lib/src/main/java/org/apache/flink/ml/common/param/HasFeatureColsDefaultAsNull.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.flink.ml.common.param;
+
+import org.apache.flink.ml.param.Param;
+import org.apache.flink.ml.param.ParamValidators;
+import org.apache.flink.ml.param.WithParams;
+
+/** Params of the names of the feature columns used for training in the input table. */
+public interface HasFeatureColsDefaultAsNull<T> extends WithParams<T> {
+    /**
+     * @cn-name 特征列名数组
+     * @cn 特征列名数组，默认全选
+     */
+    Param<String[]> FEATURE_COLS =
```

Comment:
Can we reuse classes like `StringArrayParam`? Same for other classes.

Reply:
OK, I will refine it later.

Reply:
I think we need discuss the format of features input for the algorithm. A dense vector or a string format vector or some numerical columns?

Reply:
done